src/pages/application-center/sponsorship.tsx

```diff
@@ -61,7 +61,7 @@ const Sponsorship = () => {
             </a>
             <a
               className="acp-apply-btn md:ml-5"
-              href={imageManager("prospectus_2025").pdf}
+              href={imageManager("EN_MariHacks_Sponsorship_Prospectus_2026").pdf}
```

Comment:
New prospectus looks great! Just a note that in the sponsorship Google form, the English and French prospectus links still go to last year's files, probably something that should be updated